### ISSUE TITLE
Tile Map Bug

### ISSUE THREAD

**Issue by BryanTheOverlord** [godotengine/godot#74133]
### Godot version

RC6

### System information

Windows 11

### Issue description

when i delete 1000+ tiles at once in the tile map it exits without saving and same thing when i place 1000+ tiles at once and does not save it so if you open godot and your project and you saved it 1 hr ago you just lost 1 hour of progress

### Steps to reproduce

when i delete 1000+ tiles at once in the tile map it exits without saving and same thing when i place 1000+ tiles at once and does not save it so if you open godot and your project and you saved it 1 hr ago you just lost 1 hour of progress

### Minimal reproduction project

i am using the forward+ version

**Comment by BryanTheOverlord** [comment]
yes same thing happens to me

**Comment by KoBeWi** [comment]
I don't think you can reasonably expect the editor to handle that many tiles, but I guess it shouldn't be crashing.

**Comment by Proggle** [comment]
a thousand tiles is what, a 33x33 square or something?  That doesn't seem too excessive, with 16x16 or 32x32 tiles that'd easily fit on a normal screen without zooming out

**Comment by KoBeWi** [comment]
Well you are right. The editor does not have a problem with 1000 tiles actually. When I tested I had to zoom out a lot and it crashed at maybe 100k.
I guess either the OP miscalculated or it's a hardware problem, idk. Still, it's possible to crash, so maybe something can be done to prevent this.
